packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/link-table/use-get-cell-content.ts

```diff
@@ -0,0 +1,42 @@
+import { GridCell, GridCellKind, Item } from "@glideapps/glide-data-grid";
+import { useCallback } from "react";
+import { useEntityEditor } from "../entity-editor-context";
+import { linkGridIndexes } from "./constants";
+import { LinkRow } from "./types";
+
+export const useGetCellContent = (rowData: LinkRow[]) => {
+  const { linkSort } = useEntityEditor();
+
+  const getCellContent = useCallback(
+    ([col, row]: Item): GridCell => {
+      const link = rowData[row];
+
+      if (!link) {
+        throw new Error();
+      }
+
+      const linkKey = linkGridIndexes[col];
+
+      if (!linkKey) {
+        throw new Error();
+      }
```

Comment:
Should these errors have some kind of description?

Reply:
Same https://github.com/hashintel/hash/pull/1207#discussion_r999188396